docker/vendor/grafana/include/prometheus.json

```
@@ -1023,13 +1042,104 @@
         "alignLevel": null
       }
     },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "Prometheus",
+      "description": "",
+      "fill": 1,
+      "fillGradient": 0,
+      "gridPos": {
+        "h": 7,
+        "w": 24,
+        "x": 0,
+        "y": 15
+      },
+      "id": 51,
+      "legend": {
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 1,
+      "links": [],
+      "nullPointMode": "null",
+      "options": {
+        "dataLinks": []
+      },
+      "percentage": false,
+      "pointradius": 5,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "container_memory_usage_bytes{component_name=\"prometheus\"} / container_spec_memory_limit_bytes{component_name=\"prometheus\"} * 100",
+          "format": "time_series",
+          "intervalFactor": 2,
+          "legendFormat": "{{pod_name}}",
+          "refId": "A",
+          "step": 2
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeRegions": [],
+      "timeShift": null,
+      "title": "Memory, % of limit",
```

```suggestion
      "title": "Memory Usage",
```
Just to be consistent with the other similar graphs on other dashboards.

I like to make these titles as simple as possible and elaborate in the little tooltip when you hover over the _i_ icon on the charts.

@sjmiller609 would you mind adding a tooltip part to this, that goes in line with the other ones?

Downstream enterprise customers that may not have all the information and context, so a friendly explanation can help folks figure out all these charts.

---

Actually maybe add one to the PV chart above it, while you're in there if you don't mind. Must have forgotten that one. Been trying to standardize everything across these.